React_Day8/avengers/src/index.js

```diff
@@ -0,0 +1,6 @@
+import ReactDOM from 'react-dom/client';
+import './index.css';
```

Comment:
can give an empty line between library imports and local file imports for better readibility